README.md

```diff
@@ -39,6 +40,100 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. If you haven't install Go, check it out the [docs fot it](https://golang.org/doc/install)
+#### 2. Get this repository properly cloned and installed. 
+#### 3. If you have properly setted your enviroment you will be able to run Politeia's components by simple typing its name (e.g: `politeiawwww`) on your terminal (p.s It won't fully work untill you set its configuration).
+#### 4. Finding your configuration files path: 
```

Comment:
I would merge steps 4 and 5 into one and just call it "Setup configuration files"